English (en)/Oqtane.Client/Resources/Modules/Admin/Login/Index.resx

```diff
@@ -126,4 +126,13 @@
   <data name="Forgot Password" xml:space="preserve">
     <value>Forgot Password</value>
   </data>
+  <data name="Password:" xml:space="preserve">
```

Comment:
Did you run the latest version of the resource extractor? I see some changes not related to `nl-NL`

Reply:
Commit on Apr 6, 2021 this is fixed

Reply:
I don't understand why the change happen on non `nl-NL`?

Reply:
I merge you changes, I will try to refresh the English resources later

Thanks a lot @leigh-pointer